frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -939,10 +942,13 @@
                     vm.stopLoader();
                 }
             };
-
             utilities.sendRequest(parameters);
         };
 
+        if(vm.mySubmissionPhaseSlug) {
```

Comment:
Hey @Kajol-Kumari The problem is here, actually this thing is getting called before [this](https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/js/controllers/challengeCtrl.js#L505) API but in order to work as we are using `vm.phases` inside `vm.getResults` function so the above mentioned API should call first and then `vm.getResults` function should be called.

Reply:
But @Sanji515 in the network tab i can see that [this API](https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/js/controllers/challengeCtrl.js#L504) being hit first and after that it's throwing that error of length of undefined.

Attached screenshort:
![Screenshot from 2020-05-15 10-53-13](https://user-images.githubusercontent.com/44888949/82014486-55fd1200-969a-11ea-9e16-bbfa439fc338.png)

Reply:
yes but after the execution of this(^) condition, that's reason of `vm.phases.results` is coming as `undefined`.

Reply:
yes @Sanji515, Thanks for pointing this.I just restricted the call of `vm.getResult()` only after the value is assigned in vm.phases and now it works fine for me. Please check on your end once.